incubator/fairwinds-metrics/values.yaml

```diff
@@ -36,3 +36,27 @@ controller:
     enabled: true
     type: ClusterIP
     port: 10042
+
+listener:
+  enabled: false
```

Comment:
Can you please add a ci/test-values that enables this listener so that we can e2e test it?

Reply:
👍 done